app/src/main/java/com/example/ar_reshare/AddProduct.java

```diff
@@ -168,7 +158,7 @@ public void onClick(View v) {
                         ArrayList<String> dataURIList;
                         dataURIList = convertToDataURI(adapter.uploadedImages);
                         media.addAll(dataURIList);
-                        BackendController.addProduct(productName,productDescription,"UK","Bristol",productPostcode, 1,"new", media, AddProduct.this);
+                        BackendController.addProduct(productName,productDescription,"UK","Bristol",productPostcode,category,condition, media, AddProduct.this);
```

Comment:
Hey, this is just a suggestion but when the user types in the postcode, if you use the _PostcodeHelper_ you can find the region the postcode belongs to (_so for example Bristol postcodes are "Bristol, City of"_). So maybe in the future, we could replace the hardcoded _"Bristol"_ value with a dynamically determined region? Let me know what you think!